great-idea-website/index.html

```diff
@@ -19,69 +20,102 @@
     <header>
       <nav class="navbox">
 
-          
+
         <a href="#servicesBox">Services</a>
         <a href="#productBox">Product</a>
         <a href="#visionBox">Vision</a>
         <a href="#featuresBox">Features</a>
         <a href="#aboutBox">About</a>
         <a href="#contactBox">Contact</a>
-        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-      
-        
+        <a href="#"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
```

Comment:
Liked how you wrapped an image in the <a> tag so you can click on it and it might link to a new page 👍